### PR TITLE
FAT-377040 Select "Delete items" option (Thunderjet)

### PR DESCRIPTION
**Pull request by dnovikau**
[C377040](https://foliotest.testrail.io/index.php?/cases/view/377040)
***
[Allure report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4492/allure/)
***

### CODE REVIEW

File: cypress/e2e/data-import/e2e/match-on-pol-and-update-items.cy.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ import Logs from '../../../support/fragments/data_import/logs/logs';
 import ItemRecordView from '../../../support/fragments/inventory/item/itemRecordView';
 import CheckInActions from '../../../support/fragments/check-in-actions/checkInActions';
 import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
-import OrderView from '../../../support/fragments/orders/orderView';
+import OrderDetails from '../../../support/fragments/orders/orderDetails';
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/e2e/match-on-pol-and-update-items.cy.js:31]: Run all tests and add allure Report

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js [hunk 2]
```diff
@@ -580,30 +580,6 @@ export default {
     cy.expect(paneResultsSection.exists());
   },
 
-  createInstanceViaApi() {
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventorySearchAndFilter.js:583]: Fix linter issues

File: cypress/support/fragments/orders/orderLines.js [hunk 3]
```diff
@@ -899,6 +907,14 @@ export default {
     cy.wait(4000);
   },
 
+  fillPOLineDetails({ receiptStatus }) {
+    if (receiptStatus) {
+      cy.do(poLineDetails.receiptStatus.focus());
+      cy.do(poLineDetails.receiptStatus.choose(receiptStatus));
+      // cy.expect(poLineDetails.receiptStatus.has({ value: receiptStatus }));
```
Comment (IhorBohdan) [inline cypress/support/fragments/orders/orderLines.js:914]: Does we need comment code here?

Comment (dnovikau) [reply cypress/support/fragments/orders/orderLines.js]: Fixed

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js (hunk 2 above)
Comment (dnovikau) [reply cypress/support/fragments/inventory/inventorySearchAndFilter.js:583]: Done

File: cypress/e2e/data-import/e2e/match-on-pol-and-update-items.cy.js (hunk 1 above)
Comment (dnovikau) [reply cypress/e2e/data-import/e2e/match-on-pol-and-update-items.cy.js:31]: Done

**Review by ernestyedigaryan1 — APPROVED** [review]

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**